core/src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -190,7 +190,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                                 "/user/shopping-list-items/user-shopping-list-items",
                                 "/user/shopping-list-items")
                         .hasAnyRole(USER, ADMIN, UBS_EMPLOYEE, MODERATOR, EMPLOYEE)
-                        .requestMatchers(HttpMethod.GET,
+                        .requestMatchers(HttpMethod.GET, USER_LINK,
```

Comment:
Add USER_LINK here doesn't make any sense.
I mean USER_LINK == "/user"

What is it supposed to do in this case?


In general task 251 is about POST /user/filter and it actually works fine without any changes.
I think You can just close this PR.